src/main/java/htsjdk/samtools/SamReaderFactory.java

```diff
@@ -57,6 +58,13 @@
     
     abstract public SamReader open(final File file);
 
+    public SamReader open(final Path path) {
```

Comment:
Should live in `SamReaderFactoryImpl` along with the other `open()` implementations